source/mapping-rules/calculated-fields/edit-calculated-fields.txt

```diff
@@ -1,3 +1,45 @@
+.. _rm-edit-calculated-fields:
+
 ======================
 Edit Calculated Fields
-======================
\ No newline at end of file
+======================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+You can update the JavaScript expression used in existing calcualted
+fields. Newly edited calculated fields are updated in MongoDB after your
+next sync job runs.
+
+Steps
+-----
+
+To edit a calculated field:
```

Comment:
(step intro)
It's ok to delete the step intro. It essentially repeats the title. We tried to delete them in the editorial refactor, but it looks like a few remain.

Reply:
That's my mistake, I added them myself when I did these updates. I didn't realize we were moving away from them. Will fix.